sources/Application/Views/ProjectView.cpp

```diff
@@ -119,9 +119,15 @@ ProjectView::ProjectView(GUIWindow &w,ViewData *data):FieldView(w,data) {
     v = project_->FindVariable(VAR_SOFTCLIP);
     position._y += 1;
     UIIntVarField *f3 =
-        new UIIntVarField(position, *v, "soft clip: %s", 0, 4, 1, 1);
+        new UIIntVarField(position, *v, "soft clip: %s", 0, 4, 1, 3);
```

Comment:
just FYI: initially I set it up similarly, but after testing I felt that it didn't make a lot of sense to jump between extremes here, so I left it at 1

Reply:
Yeah I agree, it's a tradeoff between being able to quickly switch between saturated and not to determine if the volume level stays the same and protecting peoples hearing. 😶
I guess preventing hearing damage should be priority.

Reply:
That makes sense, it does work as a sort of quick bypass, haven't thought of it this way

Reply:
Ideally Up/down could work as a toggle off/latest setting but that requires a new UIIntVariable among other things I guess